Remove 'proc_root' usage

Commit c74c120a21d87b0b6925ada5830d8cac21e852d9 (Linus's tree) removed
the export of 'proc_root' variable, because you can create on the proc's
root directory by passing NULL where you would pass 'proc_root'.

Signed-off-by: Luiz Fernando N. Capitulino <lcapitulino@mandriva.com.br>
Index: linux-2.6.25/3rdparty/rfswitch/av5100.c
===================================================================
--- linux-2.6.25.orig/3rdparty/rfswitch/av5100.c
+++ linux-2.6.25/3rdparty/rfswitch/av5100.c
@@ -107,7 +107,7 @@ static void av5100_proc_cleanup(void)
 {
 	if (dir_base) {
 		remove_proc_entry("radio", dir_base);
-		remove_proc_entry(DRV_NAME, &proc_root);
+		remove_proc_entry(DRV_NAME, NULL);
 		dir_base = NULL;
 	}
 }
@@ -118,7 +118,7 @@ static int av5100_proc_init(void)
 	struct proc_dir_entry *ent;
 	int err = 0;
 
-	dir_base = create_proc_entry(DRV_NAME, S_IFDIR, &proc_root);
+	dir_base = create_proc_entry(DRV_NAME, S_IFDIR, NULL);
 	if (dir_base == NULL) {
 		printk(KERN_ERR DRV_NAME ": Unable to initialise /proc/" 
 		       DRV_NAME "\n");
Index: linux-2.6.25/3rdparty/rfswitch/pbe5.c
===================================================================
--- linux-2.6.25.orig/3rdparty/rfswitch/pbe5.c
+++ linux-2.6.25/3rdparty/rfswitch/pbe5.c
@@ -138,7 +138,7 @@ static void pbe5_proc_cleanup(void)
 {
 	if (dir_base) {
 		remove_proc_entry("radio", dir_base);
-		remove_proc_entry(DRV_NAME, &proc_root);
+		remove_proc_entry(DRV_NAME, NULL);
 		dir_base = NULL;
 	}
 }
@@ -149,7 +149,7 @@ static int pbe5_proc_init(void)
 	struct proc_dir_entry *ent;
 	int err = 0;
 
-	dir_base = create_proc_entry(DRV_NAME, S_IFDIR, &proc_root);
+	dir_base = create_proc_entry(DRV_NAME, S_IFDIR, NULL);
 	if (dir_base == NULL) {
 		printk(KERN_ERR DRV_NAME ": Unable to initialise /proc/" 
 		       DRV_NAME "\n");
